plenum/common/tracker.py

```diff
@@ -0,0 +1,33 @@
+from stp_core.common.log import Logger
+
+logger = Logger()
+
+
+class Tracker:
+
+    def __init__(self):
+        self.un_committed = []
+
+    def track(self, state_root, ledger_size):
+
+        un_committed_state = ()
+
+        if state_root != "":
+            un_committed_state += (state_root,)
+        else:
+            raise logger.error("Empty state root given")
+
+        if ledger_size > 0:
+            un_committed_state += (ledger_size,)
+        else:
+            raise logger.error("Incorrect size of ledger given")
+
+        self.un_committed.append(un_committed_state)
+
```

Comment:
We should also have an API call `commit_batch()` which should remove the first value from the `un_committed`.
So, taking into account that we deal with the following 3 operation: pop left, pop right, append right, I think it's worth considering `deque` as a structure for `un_committed`.